File: C Programming/Assignments/Assignment 3 (For loop )/Q6.c

void main(){
    int n=28,fno=0;
    for(int i=1;i<n;i++){
        if(n%i==0){
            fno=fno+i;
        }
    }if(fno==n){
            printf("Perrfect");
        }else
            printf("not perfect");
    
}

File: Java/C TO JAVA/Assignment 3/Q6.java

class Q6{

public static void main (String args[]){
    int n=28,i=1;
    int fno=0;
    while(i<n){
        if(n%i==0){
            fno=fno+i;
        }
        i++;
    }if(fno==n){
        System.out.printf("Perrfect");

    }else
        System.out.printf("not perfect");
}
}